sgf-viewer/src/content.ts

```diff
@@ -61,6 +61,99 @@ export const pages: object =
                 "adversary": "34.1 million training steps, 8192 visits",
                 "victim": "Latest, 64 visits",
                 "description": ["With 64 visits, KataGo's <code>Latest</code> network plays at the level of a  <a href=\"https://arxiv.org/pdf/2211.00241.pdf#page=17\">top-20 human professional</a>. We achieve a win rate of 48% against this victim simply by increasing the number of adversary visits to 8192. The adversary wins by the same qualitative strategy of staking out a corner, but plays significantly more stones in the victim's territory. The adversary loses when the victim plays the game out to the end, resulting in a very full board."]
+            },
+            {
+                "title": "KataGo without search with pass-alive defense",
```

Comment:
For parallel structure with previous titles.
```suggestion
                "title": "Hardened KataGo without search (level of top 100 European player)",
```